style/index.css

```diff
@@ -2,3 +2,10 @@
 | Copyright (c) Jupyter Development Team.
 | Distributed under the terms of the Modified BSD License.
 |---------------------------------------------------------*/
+div.view-lines > div {
```

Comment:
This is pretty general CSS, especially with an `!important`. What is the purpose here? If we need this, it should really be narrowly targeting just monaco, and not a generic div with a `view-lines` class.